### PR TITLE
chore: Added /msg_multi for sending multiple messages & metrics for protocol loop iter latency

### PR DESCRIPTION
**Pull request by ChaoticTempest**

### CODE REVIEW

File: node/src/web/mod.rs [hunk 1]
```diff
@@ -86,7 +87,35 @@ async fn msg(
     Ok(())
 }
 
-#[derive(Serialize, Deserialize)]
+#[tracing::instrument(level = "debug", skip_all)]
+async fn msg_multi(
```
Comment (volovyks) [inline node/src/web/mod.rs:91]: Should we merge `msg` and `msg_multi` into one function?

**Review by volovyks — COMMENTED** [review]
Have you tested how it affects the protocol? Add a description, please

**Comment by ppca** [comment]
I think the send_multi seems broken given an earlier deploy on dev last night, no triples were generated. Let's not merge this until we test it on dev again.

**Comment by ChaoticTempest** [comment]
yeah let me fix this on dev. Not necessary to go to partners just yet since what we have is working just fine

**Comment by ChaoticTempest** [comment]
closing this in favor of https://github.com/near/mpc-recovery/pull/549

**Pull request closed without merging**